sw/otbn/code-snippets/randomness.s

```diff
@@ -117,13 +118,32 @@ test_urnd:
   jal x1, error_checking
   ret
 
+/**
+ * Subroutine thats consumes entropy (RND & URND)
```

Comment:
Nit: it just consumes entropy from RND which is fine (as discussed). So the comment should be updated.

Reply:
Thanks, will do!